### DISCUSSION TITLE
Start of week date picker

### DISCUSSION THREAD

**Discussion by KristerL** [carbon-design-system/carbon#7805] [Help]
I skimmed through all the issues and discussions with the keyword but could not find a solution. 

Is there any way to set the week start to Monday instead of Sunday in the date picker?

Ideally, we should be able to pass the props to flatpicker - https://flatpickr.js.org/localization/ as such:
` locale: {
        firstDayOfWeek: 2
    }`

But it seems that locale is overwritten by date picker thus this extra option cannot be passed to flatpicker: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/DatePicker/DatePicker.js#L303

Is there any other way to do this? Would prefer to use en locale but change the start of the week. Thanks!

Maybe this should be made an issue because it could be described as a bug/usability problem.

**Comment by tw15egan** [comment]
Hey there @KristerL , this does seem like a bug. It might be better to create an issue so that we can properly track this and figure out a solution for your use-case
